lambdas/service/app.py

```diff
@@ -1757,6 +1751,36 @@ def validate_wait(wait: Optional[str]) -> Optional[int]:
                                                    headers=headers)
 
 
+@app.route('/repository/sources', methods=['GET'], cors=True, method_spec={
+    'summary': 'List available data sources for the selected catalog',
```

Comment:
I don't think we use the term "selected" elsewhere. Also, what if no catalog is passed?